docs/guides/qpu-information.ipynb

```diff
@@ -0,0 +1,805 @@
+    "| Fractional gates | [`RX`](/docs/api/qiskit/qiskit.circuit.library.RXGate) (single-qubit), [`RZZ`](/docs/api/qiskit/qiskit.circuit.library.RZZGate) (two-qubit) |\n",
+    "| Non-unitary instructions | [`measure`](/docs/api/qiskit/circuit#qiskit.circuit.Measure), [`reset`](/docs/api/qiskit/circuit#qiskit.circuit.Reset) |\n",
+    "| Control flow | [`if_else`](/docs/api/qiskit/qiskit.circuit.IfElseOp) (classical feedforward) |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48cf020c-24eb-49dd-a417-7196d036b0e8",
+   "metadata": {},
+   "source": [
+    "## Dynamic backend information\n",
+    "\n",
+    "Backends can also have properties that change whenever the backed is calibrated, such as qubit frequency and operation error rates. Backend properties update after the calibration sequence completes. These properties can be used when optimizing quantum circuits or to construct noise models for a classical simulator. See the [Calibration jobs](/docs/guides/calibration-jobs) guide for more information.\n",
+    "\n",
+    "### Qubit properties\n",
+    "\n",
+    "The `backend.properties().qubit_property()` returns information about the qubits' physical attributes. It contains a dictionary of various properties of the qubit, each paired with its value and the timestamp of the last calibration.\n",
+    "\n",
+    "\n",
+    "#### `T1` (Relaxation time)\n",
+    "\n",
+    "The T1 time represents the average duration a qubit remains in its excited state $|1\\rangle$ before decaying to its ground state $|0\\rangle$ due to energy relaxation. This parameter is used to characterize the qubit's energy relaxation behavior, and is expressed in units of seconds (s).\n",
+    "\n",
+    "\n",
+    "|  |  |\n",
+    "| --- | --- |\n",
+    "| **View with Qiskit** | `backend.properties().t1(qubit)` |\n",
+    "| **View on IBM Quantum Platform** | Details section |\n",
+    "\n",
+    "#### `T2` (Dephasing time)\n",
+    "\n",
+    "The T2 time denotes the timescale over which a qubit maintains phase coherence of a superposition between the $|0\\rangle$ and $|1\\rangle$ states. It accounts for both energy relaxation and pure dephasing processes, providing insight into the qubit's coherence properties.\n",
+    "\n",
+    "\n",
+    "|  |  |\n",
+    "| --- | --- |\n",
+    "| **View with Qiskit** | `backend.properties().t2(qubit)` |\n",
+    "| **View on IBM Quantum Platform** | Details section |\n",
+    "\n",
+    "#### `frequency`\n",
+    "\n",
+    "This parameter specifies the resonant frequency of the qubit, indicating the energy difference between the $|0\\rangle$ and $|1\\rangle$ states, expressed in hertz (Hz).\n",
+    "\n",
+    "\n",
+    "|  |  |\n",
+    "| --- | --- |\n",
+    "| **View with Qiskit** | `backend.properties().frequency(qubit)` |\n",
+    "| **View on IBM Quantum Platform** | Calibration data section, graph view  |\n",
+    "\n",
+    "#### `anharmonicity`\n",
```

Comment:
I'll need to investigate more about frequency and anharmonicity, to see if they are no longer reported or is just broken in qiskit-ibm-runtime.